rsrc/rsrc.go

```diff
@@ -13,95 +13,89 @@ import (
 	"github.com/akavel/rsrc/internal"
 )
 
-const (
-	RT_ICON       = coff.RT_ICON
-	RT_GROUP_ICON = coff.RT_GROUP_ICON
-	RT_MANIFEST   = coff.RT_MANIFEST
-)
-
 // on storing icons, see: http://blogs.msdn.com/b/oldnewthing/archive/2012/07/20/10331787.aspx
-type GRPICONDIR struct {
+type _GRPICONDIR struct {
 	ico.ICONDIR
-	Entries []GRPICONDIRENTRY
+	Entries []_GRPICONDIRENTRY
 }
 
-func (group GRPICONDIR) Size() int64 {
+func (group _GRPICONDIR) Size() int64 {
 	return int64(binary.Size(group.ICONDIR) + len(group.Entries)*binary.Size(group.Entries[0]))
 }
 
-type GRPICONDIRENTRY struct {
+type _GRPICONDIRENTRY struct {
 	ico.IconDirEntryCommon
 	Id uint16
 }
 
 func Embed(fnameout, arch, fnamein, fnameico string) error {
-	newid := make(chan uint16)
-	go func() {
-		for i := uint16(1); ; i++ {
-			newid <- i
-		}
-	}()
+	lastid := uint16(0)
+	newid := func() uint16 {
+		lastid++
+		return lastid
+	}
 
-	coff := coff.NewRSRC()
-	err := coff.Arch(arch)
+	c := coff.NewRSRC()
```

Comment:
please rename `c` to `out` (argh, now I see why I did this weird reexport of consts)